global-cli/index.js

```diff
@@ -171,7 +171,7 @@ function checkNodeVersion() {
 // https://github.com/facebookincubator/create-react-app/pull/368#issuecomment-237875655
 function isGitHubBoilerplate(root) {
```

Comment:
Let’s rename this to `isSafeToCreateProjectIn(root)`.
Then add comment like

```js
// If project only contains files generated by GH, it’s safe.
// We also special case WebStorm .idea because it integrates with CRA:
// https://github.com/facebookincubator/create-react-app/pull/368#issuecomment-243446094
```